src/deploy/functions/services/index.ts

```diff
@@ -30,12 +33,19 @@ export const PubSubService: Service = {
   ensureTriggerRegion: noop,
 };
 /** A storage service object */
-export const StorageService = {
+export const StorageService: Service = {
   name: "storage",
   api: "storage.googleapis.com",
   requiredProjectBindings: obtainStorageBindings,
   ensureTriggerRegion: ensureStorageTriggerRegion,
 };
+/** A firebase alerts service object */
+export const FirebaseAlertsService: Service = {
+  name: "firebasealerts",
+  api: "logging.googleapis.com",
```

Comment:
This is really the API for Firebase Alerts?? How is this not a conflict with Cloud Audit Logs?

Reply:
This was just a placeholder, I'll update it with the correct API. Additionally, this field is not currently being used in the CLI